velox/functions/prestosql/TypeOf.cpp

```diff
@@ -93,6 +91,9 @@ std::string typeName(const TypePtr& type) {
           typeName(type->childAt(0)),
           typeName(type->childAt(1)));
     case TypeKind::ROW: {
+      if (isIPPrefixType(type)) {
```

Comment:
@mohsaka : The change in this file should also be submit as an individual PR.

Reply:
https://github.com/facebookincubator/velox/pull/11449